frontend/src/components/maps/leaflet/LayerPopup/hooks/useBoundaryLayer.tsx

```diff
@@ -1,9 +1,30 @@
 import { useEnvironment } from 'hooks/useEnvironment';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+
+export const Username = () => {
+  const keycloak = useKeycloakWrapper();
+  return keycloak.username;
+};
+
+export const FindUserType = (username: string) => {
```

Comment:
This function looks like it can return void too.  Not sure if that's desirable.

Reply:
What would you recommend to make this cleaner?

Reply:
Generally it's bad practice to return void, or undefined.  It's even worse practice to return inconsistent results.  Since you're returning a string value, you should return an empty string if no match is found "".